dev/TimePicker/TimePicker_themeresources.xaml

```diff
@@ -8,6 +8,8 @@
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Default">
             <x:Double x:Key="TimePickerSelectorThemeMinWidth">80</x:Double>
+            <x:Double x:Key="TimePickerSpacerThemeWidth">2</x:Double>
```

Comment:
I think the spacer also needs to be 1px?

![20190802171347](https://user-images.githubusercontent.com/4424330/62404433-128d4900-b549-11e9-9ebc-f382be4b3d7c.png)